### PR TITLE
ReactJS-week1-Amanda&Jessica 

### PR DESCRIPTION
**Pull request by jessnmo**
Link to my deployed project:

My reflections on how this week's project turned out:

Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

File: code/src/components/Name.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+export const Name = (props) => {
+    {}
+    
+
+}
```
Comment (KajaWilbik) [inline code/src/components/Name.js:7]: Are you using this component? If yes what is it doing?

**Review by KajaWilbik — APPROVED** [review]
Nice job. The website works perfectly and it is fully responsive. Great job with the comments, it helped to understand your code.

File: code/src/App.js [hunk 2]
```diff
@@ -1,12 +1,26 @@
...
+      <section className="main-container">
+      <div className='header'> <h1> New Release </h1> </div> 
+      
+      <div className='album-container'>
+        {data.albums.items.map((album) => {
+          return <Album key={album.id} album={album} />;
+        })}
+      </div>
+      </section>  
+    </>
```
Comment (Bibiruu) [inline code/src/App.js:24]: Before I start, How amazing that  you submitted the project like half way through the week! I like the approach of the code here. This code part is easy to read. Nice ,simple display to be deployed to the html.

File: code/src/components/Album.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
+import React from "react";
+
+
+export const Album = (props) => {
+    
+ 
+    //return the album cover 
+    return (
+        <>
```
Comment (Bibiruu) [inline code/src/components/Album.js:9]: For this part you can use in the future an option of 

<React.Fragment>  

the above example can also store a key value as well, when you map before it.

File: code/src/components/Album.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+         {/* Return artist name*/ }
+                <div className="artistName">
+                    {props.album.artists.map((artist, index) => {
+                        return (
+                        <a href={artist.external_urls.spotify}
+                        target="_blank"
+                        key={artist.id}
+                        className="nameArtist" alt="link to spotify">
+                        <p className="artist"> {artist.name}
+                        {generateDelimiter(index, props.album.artists.length)}</p>
```
Comment (Bibiruu) [inline code/src/components/Album.js:46]: Amazing how this was thought of. Using an inbuilt function to store a value. Very creative. I quick read about this, did you import this through npm install? Cool!

File: code/src/components/Album.js [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+            </div>
+        </>
+    )
+} 
+/* To add "," and "&" in the artist name */
+    const generateDelimiter = (index, length) => {
+        if (index === length - 2) {
+        return " & ";
+        } else if (index < length - 2) {
+        return ", ";
```
Comment (Bibiruu) [inline code/src/components/Album.js:60]: Learnt something new in return an else with a ; only

**Review by Bibiruu — COMMENTED** [review]
I really like your code you guys made. clean, simple and straight to the point. Nice use of components in an effective way. Overall pair job when I saw that you guys submitted 4 days ago is GOALS. Good job and congrats!

**Review by Bibiruu — COMMENTED** [review]
I really like your code you guys made. clean, simple and straight to the point. Nice use of components in an effective way. Overall pair job when I saw that you guys submitted 4 days ago is GOALS. Good job and congrats!

**Pull request closed without merging**